### PR TITLE
fix(template_helpers) :: stop plus and minus from overflowing on large integers

### PR DESCRIPTION
**Pull request by 81reap**
Both helpers did unchecked `i64` math, so large numbers panicked the renter in debug or silently overflowed in release. Now helpers fall back to floating point.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
  - Chart data rows can set their own `color`, painting a single bar, slice or point instead of the whole series. It applies to `bar`, `column`, `rangeBar`, `pie`, `treemap`, `scatter` and `bubble` charts, and to the markers of a `line` or an `area` chart.
 - Updated the bundled Tabler icon sprite from v3.44.0 to [v3.46.0](https://github.com/tabler/tabler-icons/releases/tag/v3.46.0). This adds 18 icons, including `play-bug`, `remote-control`, `rocking-chair`, `run-sprint`, `tabs`, `treasure-chest`, `vault`, and `yarn`, and includes the upstream fixes listed in the release notes.
 - Recursive `sqlpage.run_sql` errors now show one concise, positioned error instead of repeating the full inclusion chain.
+- The `plus` and `minus` template helpers no longer overflow on large integers.
```
Comment (lovasoa) [inline CHANGELOG.md]: you can't add lines to the changelog of a version that has already been released !

File: src/template_helpers.rs [hunk 2]
```diff
@@ -744,4 +740,153 @@ mod tests {
     fn as_json_context<'a>(path: &'a str, value: &'a Value) -> ScopedJson<'a> {
         ScopedJson::Context(value, vec![path.to_string()])
     }
+
+    mod documented_helpers {
```
Comment (lovasoa) [inline src/template_helpers.rs:755]: most of these already passed before, right ?

**Review by lovasoa — CHANGES_REQUESTED** [review]
Ideally one pr should address one specific point. If the stated goal is to fix a bug in one particular edge case, then add a test for that particular edge case.

File: CHANGELOG.md (hunk 1 above)
Comment (81reap) [reply CHANGELOG.md]: fixed!

File: src/template_helpers.rs (hunk 2 above)
Comment (81reap) [reply src/template_helpers.rs:755]: yep, only prod fails on `integer_arithmetic_falls_back_to_floats_instead_of_overflowing` which also handles

> If the stated goal is to fix a bug in one particular edge case, then add a test for that particular edge case.

**Comment by 81reap** [comment]
> Ideally one pr should address one specific point.

the `h0` change seemed too small to PR on it's own and didn't really fit into the stack. I raised it as a separate fix here #1419

**Review by lovasoa — APPROVED** [review]
